src/server/pachyderm_test.go

```diff
@@ -126,10 +125,10 @@ func TestDuplicatedJob(t *testing.T) {
 		}},
 	}
 
-	job1, err := pachClient.CreateJob(context.Background(), req)
+	job1, err := c.PpsAPIClient.CreateJob(context.Background(), req)
```

Comment:
So what's going on here is that `c.CreateJob` and `c.PpsAPIClient.CreateJob` are actually different methods. `c.CreateJob` is the nice client method that we define and that is documented. `c.PpsAPIClient.CreateJob` is the grpc call. In a lot of places in our tests we still need to punch through to the grpc method because `c.CreateJob` doesn't expose all the functionality of the underlying method.

I agree with your point in general though so over time we should expand the functionality of the client methods so you can do everything with them and then use them in tests instead of the grpc methods so that tests can server as documentation.